src/index.js

```diff
@@ -0,0 +1,65 @@
+import './style.css';
+
+const taskForm = document.getElementById('task-form');
+const taskInput = document.getElementById('task-input');
+const taskList = document.getElementById('task-list');
+const deleteSelectedButton = document.getElementById('delete-selected');
+
+function saveTasksToLocalStorage() {
+  const tasks = [];
+  const taskItems = taskList.querySelectorAll('li');
+  taskItems.forEach((taskItem) => {
+    const checkbox = taskItem.querySelector("input[type='checkbox']");
+    const text = taskItem.querySelector('span').textContent;
+    const completed = checkbox.checked;
+    tasks.push({ text, completed });
+  });
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+// save data on local storage
```

Comment:
- [ ] You've done extra work on this milestone 💯 However, there's a very small task left to do in order to pass this milestone. Please hardcode an array of tasks and let the DOM display them dynamically on the page load. 🙏 

```javascript
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
  {
  ...
  },
];
```